services/frontend-service/src/ui/components/SideBar/SideBar.tsx

```diff
@@ -372,6 +385,76 @@ export const SideBar: React.FC<{ className?: string; toggleSidebar: () => void }
         () => actions.length > 0 && (!newLockExists || lockMessage),
         [actions.length, lockMessage, newLockExists]
     );
+    const appLocksRendered =
+        conflictingLocks.appLocks.length === 0 ? undefined : (
+            <>
+                <h4>Conflicting App Locks:</h4>
+                <ul>
+                    {conflictingLocks.appLocks.map((appLock: DisplayLock) => (
+                        <li>
+                            <DisplayLockInlineRenderer
+                                lock={appLock}
+                                key={appLock.lockId + '-' + appLock.application + '-' + appLock.environment}
+                            />
+                        </li>
+                    ))}
+                </ul>
+            </>
+        );
+    const envLocksRendered =
+        conflictingLocks.environmentLocks.length === 0 ? undefined : (
+            <>
+                <h4>Conflicting Environment Locks:</h4>
+                <ul>
+                    {conflictingLocks.environmentLocks.map((envLock: DisplayLock) => (
+                        <li>
+                            <DisplayLockInlineRenderer
+                                lock={envLock}
+                                key={envLock.lockId + '-' + envLock.environment}
+                            />
+                        </li>
+                    ))}
+                </ul>
+            </>
+        );
+    const confirmationDialog: JSX.Element = hasLocks ? (
+        <div
+            className={
+                'confirmation-dialog-container ' +
+                (dialogState.showConfirmationDialog ? 'confirmation-dialog-container-open' : '')
+            }>
+            <ConfirmationDialog
+                headerLabel={'Please Confirm the Deployment over Locks'}
+                onConfirm={applyActions}
+                confirmLabel={'Confirm Deployment'}
+                onCancel={cancelConfirmation}
+                open={dialogState.showConfirmationDialog}>
+                <div>
+                    You are attempting to deploy apps, although there are locks present. Please check the locks and be
+                    sure you really want to ignore them.
+                    <div className={'locks'}>
+                        {appLocksRendered}
```

Comment:
In the locks page, we display the environment locks first and then the application locks. I think we should display the environment locks first and then application locks for consistency

Reply:
done